source/expressions.tex

```diff
@@ -884,8 +884,7 @@
 
 \pnum
 \indextext{conversion!bool@\tcode{bool}}%
-If the destination type is \tcode{bool}, see~\ref{conv.bool}. If the
```

Comment:
What exactly is the rationale for removing this? There seems to be unresolved overlap/conflict between [conv.integral] and [conv.bool].

Reply:
Ah, you exclude boolean conversions below. Ok.